### PR TITLE
Treepaging

### PR DESCRIPTION
**Pull request by BlackbitNeueMedien**
Hello @weisswurstkanone,
I put the fix in and checked. But the problem still seems to occur.
I saw that our customer used quotation marks for some keys in his object hierarchy. So I took a look at the code myself and found the following fix, which also works. I ask you to check my suggested solution.

### CODE REVIEW

**Comment by weisswurstkanone** [comment]
Can you please fix the tests?

Fatal error: Cannot redeclare Pimcore\Model\Element\Service::getSortIndexPath() in /home/travis/build/pimcore/pimcore/models/Element/Service.php on line 169

+ sign the CLA

File: models/Element/Service.php [hunk 1]
```diff
@@ -117,6 +117,40 @@ public static function getTypePath($element)
...
+     * @static
+     * @internal
+     *
+     * @param DataObject|Document $element
+     *
+     * @return string
+     *
+     * @throws \Exception
+     */
+    public static function getSortIndexPath($element)
```
Comment (weisswurstkanone) [inline models/Element/Service.php:131]: Which branch is this based?

Comment (BlackbitNeueMedien) [reply models/Element/Service.php:131]: I have pulled and merged the pimcore/master with our forked master branch. After that I had checkout the branch treenpaging. Or should I create a new pull request based from master?

Comment (weisswurstkanone) [reply models/Element/Service.php:131]: please use the master as a basis

Comment (BlackbitNeueMedien) [reply models/Element/Service.php:131]: Okay, I had done.

**Comment by BlackbitNeueMedien** [comment]
Transfered to #6526

**Pull request closed without merging**